Logger/BaseLogger.cs

```diff
@@ -2,6 +2,11 @@
 {
     public abstract class BaseLogger
     {
+        public string className
```

Comment:
Properties should be named with PascalCase
```suggestion
        public string ClassName
```